Bluepill-cli/Bluepill-cli/Bluepill/Bluepill.m

```diff
@@ -240,8 +239,10 @@ - (void)createSimulatorWithContext:(BPExecutionContext *)context {
         // If we failed to create the simulator, there's no reason for us to try to delete it, which can just cause more issues
         if (--__self.maxCreateTries > 0) {
```

Comment:
this `maxCreateTries` was actually some temp code to debug a simulator creation issue in travis. We should use the global retry count and fall back on `- (void)proceed` to retry